frameworks/Crystal/kemalyst/config/database.yml

```diff
@@ -0,0 +1,2 @@
+pg:
+  database: postgres://benchmarkdbuser:benchmarkdbpass@127.0.0.1/hello_world
```

Comment:
you can replace `127.0.0.1` with `TFB-database` which allows you to remove the `sed` statement in the `setup.sh`